index.bs

```diff
@@ -292,20 +292,27 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl/
       of the indicated <a>background operation</a>.
     </p>
     <p>
-      When invoked, it MUST return <a>a new promise</a> <var>promise</var> and run the following
-      steps <a>in parallel</a>:
+      When invoked, it MUST run the following steps:
     </p>
     <ol>
       <li>
+        Let <var>promise</var> be <a>a new promise</a>.
+      </li>
+      <li>
         Let <var>origin</var> be the <a>entry settings object</a>'s <a>origin</a>.
       </li>
       <li>
         If the <var>origin</var> is not a <a>secure context</a>, reject <var>promise</var> with a
         <code>{{SecurityError}}</code> and terminate these steps.
```

Comment:
Unrelated to the problem I reported, but in this code path the promise isn't actually returned. s/and terminate these steps/and return promise/. The new deal in https://infra.spec.whatwg.org/#algorithm-control-flow is that "return" implies "terminate these steps".